### PR TITLE
Add windows installation

### PR DESCRIPTION
**Pull request by mabelzhang**
Partially addresses https://github.com/ignitionrobotics/docs/issues/117

I haven't added a tutorial page to address this other issue https://github.com/ignitionrobotics/docs/issues/14#issuecomment-750431388 , since ign-tools is "tricky"? Do we want a tutorial page for it?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,19 +19,64 @@ source:
...
+
 Standard installation can be performed in UNIX systems using the following
 steps:
 
- - mkdir build/
- - cd build/
- - cmake .. -DCMAKE_INSTALL_PREFIX=/usr
- - sudo make install
+```
+mkdir build/
```
Comment (JShep1) [inline README.md:48]: `build/` -> `build` and all below directories, just to keep consistency with the other tutorials

**Review by JShep1 — COMMENTED** [review]
Small nit, also we should likely include the binary installation for windows, the install command is `conda install libignition-tools --channel conda-forge`

Also, do you know how to use the actual command line tooling?  I found `ign` residing in my conda environment's `Library/bin` but it doesn't seem to be executable.  I'll try to look into this more next week.

**Comment by mabelzhang** [comment]
This library's tutorial is one of the very different ones from the rest... It doesn't have a binary install section at the moment. If we add binary instruction for Windows, we should add it for Ubuntu too, else it'd look like we don't have a Ubuntu binary package for it. I wasn't sure whether to just revamp the whole thing or follow what's there.

This week, it sounds like we're gonna try to make them all the same, so I went in and did some shelving in efbcf3c, to make it consistent with the formatting in `ign-common` and `ign-math`. Since there's so much diff already, I thought I might as well run `dos2unix` on it, because the end lines looked very weird in diffs in the terminal. So... GitHub is gonna mark the whole file as different.

I haven't tried the `ign` command either.

**Comment by mabelzhang** [comment]
I'm going to close and reopen a PR. The rebasing onto Citadel has too many conflicts.

**Comment by mabelzhang** [comment]
I replaced the branch so I can't reopen this PR... Opening a 3rd.

**Pull request closed without merging**